easybuild/scripts/bootstrap_eb.py

```diff
@@ -469,13 +472,13 @@ def stage2(tmpdir, templates, install_path, distribute_egg_dir, sourcepath):
     handle.write(EASYBUILD_EASYCONFIG_TEMPLATE % templates)
     handle.close()
 
-    # unset $MODULEPATH, we don't care about already installed modules
-    os.environ['MODULEPATH'] = ''
```

Comment:
why what? not reset it?

because
* i) it's not needed (I can't think of a reason why wiping `$MODULEPATH` would matter, other than having `EasyBuild` module available already)
* ii) wrong, especially in combination with Lmod (since it may trigger unloading of modules because they're deemed inactive because they're not longer available)